mozjs/js/public/StructuredClone.h

```diff
@@ -29,8 +29,10 @@ enum TransferableOwnership {
     // Structured clone buffer does not yet own the data
     SCTAG_TMO_UNOWNED = 1,
 
+#ifndef RUST_BINDGEN
```

Comment:
It'd be good to have comments on stuff like this. We can do that during upstreaming, though.